src/main/java/org/truffleruby/core/method/MethodNodes.java

```diff
@@ -213,43 +215,57 @@ public DynamicObject unbind(VirtualFrame frame, DynamicObject method) {
     @CoreMethod(names = "to_proc")
     public abstract static class ToProcNode extends CoreMethodArrayArgumentsNode {
 
+        private static final FrameDescriptor descriptor = new FrameDescriptor();
```

Comment:
Ruby FrameDescriptor should always have `nil` as default.
Sharing it is only OK if no slot is ever added to it, is that the case?

Reply:
I believe no slot will be added.